packages/components/src/components/as-category-widget/as-category-widget.tsx

```diff
@@ -51,6 +51,14 @@ export class CategoryWidget {
    */
   @Prop() public disableInteractivity: boolean = false;
 
+  /**
+   * If this property receives a function, it will be used to format the numbers (eg. for adding $ or €).
```

Comment:
Function used to format the number output. (e.g. adding currency 10€ or 10$)

Reply:
👌